example/apps/example-app/translations/src/en/validation.json

```diff
@@ -26,6 +26,10 @@
   "countryOfHearing": {
     "default": "Select where the appeal hearing is to be held"
   },
+  "countrySelect": {
+    "default": "Enter a valid country of residence",
+    "required": "Enter your country of residence"
+  },
```

Comment:
Had a look at GRO and they use the following
`"countrySelect": {
    "default": "Enter your country",
    "equal": "Pick a country from the list"
  }`